ccla-managers-and-ccla-signatories/whitelist-contributors.md

```diff
@@ -1,15 +1,15 @@
-# Whitelist Contributors
+# Approved List Contributors
 
-Whitelists are lists of domain names, email addresses of individuals, GitHub usernames, or GitHub organization names who are authorized to contribute under a signed Corporate CLA. As a CCLA manager, you allow contributions to projects on behalf of your company by using any whitelist:
+Whitelists are lists of domain names, email addresses of individuals, GitHub usernames, or GitHub organization names who are authorized to contribute under a signed Corporate CLA. As a CCLA manager, you allow contributions to projects on behalf of your company by using any Approved List:
 
-* Domain Whitelist allows entities to contribute under any email address under that domain name.
-* Email Whitelist allows entities to contribute under an individual email address.
-* GitHub Whitelist allows entities to contribute under a GitHub username.
-* GitHub Organization Whitelist allows entities to contribute under a GitHub organization name.
+* Domain Approved List allows entities to contribute under any email address under that domain name.
+* Email Approved List allows entities to contribute under an individual email address.
+* GitHub Approved List allows entities to contribute under a GitHub username.
+* GitHub Organization Approved List allows entities to contribute under a GitHub organization name.
 
-Each whitelist applies to the project for which the company has signed a Corporate CLA. The CLA application checks all the whitelists for allowing contributions to a company project. A contributor only needs to be on one whitelist. Contributors can use EasyCLA to send email requests to be associated \(whitelisted\) with the company.
+Each Approved List applies to the project for which the company has signed a Corporate CLA. The CLA application checks all the whitelists for allowing contributions to a company project. A contributor only needs to be on one Approved List. Contributors can use EasyCLA to send email requests to be associated \(whitelisted\) with the company.
 
-_Multiple CCLA managers cannot whitelist the same domain and sign a CCLA for the same company._
+_Multiple CCLA managers cannot Approved List the same domain and sign a CCLA for the same company._
```

Comment:
_Multiple CCLA managers cannot add to the Approved List the same domain and...

Reply:
Solved